lib/configManager.js

```diff
@@ -1,9 +1,10 @@
 const path = require('path')
 const yaml = require('js-yaml')
 module.exports = class ConfigManager {
-  constructor (context, ref) {
+  constructor (context, ref, log) {
     this.context = context
     this.ref = ref
+    this.log = log
```

Comment:
Wouldn't it be possible to use the logger through the `context`?
```suggestion
    this.log = context.log
```

Reply:
Ah, great! I wasn't aware of that. I added this because of null references for `this.log` when trying to run the tests in index.test.js. I'll push a change.

Reply:
Also, good catch, I obviously did not update all the places where ConfigManager were created in index.js! Fixup commit pushed (will rebase when review is done)